depz/depz.py

```diff
@@ -1235,11 +1243,16 @@ def check (branch, compares):
         if ahead==0 and behind==0: continue
         if behind>0: all_current=False
         if ahead>0 and behind>0:
-          llog.warn("Current branch %s is %s commits behind %s (and %s ahead)", branch, behind, c, ahead)
+          llog.warn("Current branch %s is %s behind %s (and %s ahead)", branch, App.pluralize(behind,"commit"), c, ahead)
+          llog.warn("It appears the repository is in a conflicted state with its origin.")
+          llog.warn("If the repo has local changes, manually merge or otherwise resynchronize it.")
+          llog.warn("If the repo doesn't have local changes you can delete it and run 'depz --init' to restore it.")
```

Comment:
Suggestion-style log messages as above.

(I'll stop saying this here; applies to things below.)